README.md

```diff
@@ -31,41 +29,46 @@ plasma_cli create account
 ## Deposit ETH into the OMG Network
 
 ```
-plasma_cli deposit --privatekey=<private key> --client=<local rpc server or Infura URL> --contract=<contract address> --amount=<amount in wei> --owner=<public key of the owner>
+plasma_cli deposit --privatekey="private_key" --client="local_rpc_server_or_Infura_URL" --contract="contract_address" --amount=amount_in_wei --owner="owner_address"
 ```
 
 
 ## Retrieve a List of UTXOs
 
 ```
-plasma_cli get utxos --watcher=<watcher URL> --address=<public_address>
+plasma_cli get utxos --watcher="watcher_URL" --address="public_address"
 ```
 
 ## Check the Balance of an Account
 
 ```
-plasma_cli get balance --watcher=<watcher URL> --address=<public_address>
+plasma_cli get balance --watcher="watcher_URL" --address="public_address"
 ```
-
 ## Sending ETH UTXO
 
-This function will either
-1. Send the entire UTXO to an address, or
-2. Split the UTXO to destination address and send the change to the owner
+Send an entire UTXO to an address, sends change back to sender if available
+
+```
+plasma_cli send --fromutxo=UTXO_position --fromowner="from_address" --privatekey="from_privatekey" --toowner="to_address" --toamount=to_amount  --watcher="watcher_url"
```

Comment:
`--fromowner` can go, we can derive from the private key

Reply:
just checked, yeah this is quite easy to derive, I will remove the --fromowner flags from all the tx function :)